### PR TITLE
add an endpoint to list admission plugins

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**: add an endpoint to list admission plugins

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

```release-note
add an endpoint to list admission plugins: GET /api/v1/admission/plugins/{version}
```

### CODE REVIEW

File: api/pkg/handler/routes_v1.go [hunk 1]
```diff
@@ -559,6 +560,12 @@ func (r Routing) RegisterV1(mux *mux.Router, metrics common.ServerMetrics) {
 	mux.Methods(http.MethodGet).
 		Path("/labels/system").
 		Handler(r.listSystemLabels())
+
+	//
```
Comment (floreks) [inline api/pkg/handler/routes_v1.go:564]: Leftover?

Comment (zreigz) [reply api/pkg/handler/routes_v1.go:564]: it's everywhere like this

Comment (floreks) [reply api/pkg/handler/routes_v1.go:564]: That's a bit weird but ok.

**Pull request merged**